src/components/Chat/Chat.js

```diff
@@ -63,7 +63,7 @@ export default function Chat({ showChat, toggleChat }) {
 
   return showChat ? (
     <aside className="chat">
-      <button onClick={toggleChat} className="close-chat">
+      <button onClick={toggleChat} className="close-chat" type="button">
```

Comment:
I'm surprised we need `type="button"` here 🤔 If it's some sort of a11y rule just having the `button` tag should be enough. Not a big deal either way

Reply:
Nevermind, today I learned: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/button-has-type.md